lxd/cluster/config/config.go

```diff
@@ -248,6 +248,23 @@ func (c *Config) OIDCServer() (string, string, string) {
 	return c.m.GetString("oidc.issuer"), c.m.GetString("oidc.client.id"), c.m.GetString("oidc.audience")
 }
 
+// ClusterHealing returns the configured healing threshold, i.e. the
+// number of seconds after which an offline node will be evacuated automatically. If the config key
+// is set but its value is lower than cluster.offline_threshold it returns
```

Comment:
> If the config key is set but its value is lower than cluster.offline_threshold it returns the value of cluster.offline_threshold instead.

This comment isn't fully correct because if its set to zero (or not set) then it returns zero which is not the value of `cluster.offline_threshold`. This detail is pretty important because (I assume) setting it to zero disables the feature.